build.sbt

```diff
@@ -136,6 +146,39 @@ lazy val `google-client` = crossProject(JVMPlatform)
     mimaBinaryIssueFilters := Nil
   )
 
+lazy val `http4s-client` = crossProject(JVMPlatform)
+  .withoutSuffixFor(JVMPlatform)
+  .crossType(CrossType.Pure)
+  .in(file("http4s-client"))
+  .dependsOn(core)
+  .settings(commonSettings)
+  .settings(
+    resolvers ++= Resolver.sonatypeOssRepos("snapshots"),
+    name := "bigquery-http4s-client",
+    libraryDependencies ++= {
+      val binaryVersion = scalaBinaryVersion.value
+      Seq(
+        "org.scalameta" %% "munit-scalacheck" % "0.7.29" % Test,
+        "org.typelevel" %% "munit-cats-effect-3" % "1.0.7" % Test,
+        ("io.chrisdavenport" %% "http4s-grpc-google-cloud-bigquerystorage-v1" % "3.0.0+0.0.6")
+          .exclude("io.chrisdavenport", s"http4s-grpc_${binaryVersion}"),
+        ("io.chrisdavenport" %% "http4s-grpc" % "0.0.4")
```

Comment:
this has now been moved to the http4s organization. We should get a new release of that before merge